### DISCUSSION TITLE
Why can't I run examples with my installed version of `perseus`?

### DISCUSSION THREAD

**Discussion by novvoo** [framesurge/perseus#178] [General]
>git clone https://github.com/arctic-hen7/perseus.git  ; build release perseus
>cd perseus/examples/demos/auth; perseus serve 
> some error report 
```
error[E0425]: cannot find value `dflt_server` in crate `perseus_integration`
 --> examples/demos/auth/src/main.rs:7:38
  |
7 | #[perseus::main(perseus_integration::dflt_server)]
  |                                      ^^^^^^^^^^^ not found in `perseus_integration`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `perseus-example-auth` due to previous error
```

**Comment by arctic-hen7** [comment]
The `perseus` CLI you have installed does not match the version of the main repository, so you need to use the bleeding edge version of the CLI. You can do this with `bonnie dev example demos auth serve` in your case. Make sure you run `bonnie setup` first. (You can install Bonnie with `cargo install bonnie`, and you'll also need `rust-script`.)

The reason for this is that the main repo makes all its examples, including the `examples/demos/`, generic over server integrations, which means feature flags need to be provided to make everything work. This is automatically performed by the Bonnie scripts in the repo.

**Reply by novvoo** [reply]
thx,🍺
- use bonnie:
```
cargo install bonnie rust-script
bonnie setup
bonnie dev example demos auth serve
```

- use perseus 
```
git clone https://github.com/arctic-hen7/perseus.git
cd perseus/examples/demos/auth
perseus serve '--cargo-engine-args=--features "perseus-integration/warp" ' -w
```

**Reply by arctic-hen7** [reply]
Correct, but the second method will only work sometimes. If a breaking change is added in the latest commit, that will fail.
